### DISCUSSION TITLE
Dropdown component to build menus

### DISCUSSION THREAD

**Discussion by micebrain** [Budibase/budibase#3292] [Help]
Hi,
I am trying to build a new layout with drop down navigation. There is no such component, and I was trying to use the <embed> component with Bootstrap. While the css works, the drop down effect does not as it requires Bootstrap js which is not supported at the moment.
Is there a way to get around this?
Thanks!

**Comment by mjashanks** [comment]
I can't think of a way to do this. Probably not possible! 

Why do you want a dropdown nav? Something to do with display on mobile?  Or do you just want to categorize your nav items?

The existing Navs should support mobile, so I'd be interested to know your use case here.

**Reply by micebrain** [reply]
It's not necessary just for mobile. But the ability to group related items together to keep the nav cleaner. 
Currently the nav is only one level, it works well if there are only few links. With more links, this becomes quite difficult to manage.
As an example here from Slack:

![Screenshot 2021-11-09 at 09 10 32](https://user-images.githubusercontent.com/7136209/140886460-023e0dd0-3208-440a-97b5-850c796e4e80.png)
